src/components/Form/MultipleDropdown/MultipleDropdown.jsx

```diff
@@ -6,16 +6,25 @@ export default class MultipleDropdown extends ValidationElement {
   constructor (props) {
     super(props)
 
+    this.state = {
+      uid: `${this.props.name}-${super.guid()}`
+    }
+
     this.remove = this.remove.bind(this)
     this.update = this.update.bind(this)
     this.updateToken = this.updateToken.bind(this)
+    this.handleError = this.handleError.bind(this)
   }
 
   update (queue) {
     this.props.onUpdate({
       value: this.props.value,
       ...queue
     })
+
+    setTimeout(() => {
+      this.handleError(queue, [])
+    }, 1000)
```

Comment:
Hmm, what's the story here? Is there a reason why this handleError can't get triggered immediately based off an event vs a timeout? Best to avoid waiting for errors (1s might not always be enough as well). Might not be reading this right though, let me know if you think it requires action.

Reply:
Went ahead and removed in https://github.com/truetandem/e-QIP-prototype/pull/2276 after testing in browser and test suite passes.

Reply:
Great thanks! Could you issue that commit to this PR so that it can be merged? I think it's jst on the truetandem fork right now.